### PR TITLE
 WIP - Add Ensign for Data Engineers Page

### PR DESCRIPTION
**Pull request by pdamodaran**
### Scope of changes

I have added documentation on how to use Ensign and Watermill to pull data from a Weather API and load into a PostgreSQL database.

There are a few TODOs
- [x] The code currently works without an Ensign API Key.  I need to update the code and documentation to use an Ensign API Key.
- [x] I need write access to [ensign-examples](https://github.com/rotationalio/ensign-examples) so that I can add the code there update the full code link in the documentation.
- [x] I have two png files that are not rendering.  They are currently under `docs/examples`.  I need to move them to the appropriate location.
- [x] I need write access to [watermill-ensign](https://github.com/rotationalio/watermill-ensign) so that I can add a code example.

Fixes SC-12538

### Type of change

- [ ] new feature
- [ ] bug fix
- [x] documentation
- [ ] testing
- [ ] technical debt
- [ ] other (describe)

### Acceptance criteria

Describe how reviewers can test this change to be sure that it works correctly. Add a checklist if possible.

### Author checklist

- [ ] I have manually tested the change and/or added automation in the form of unit tests or integration tests
- [ ]  I have updated the dependencies list
- [ ]  I have recompiled and included new protocol buffers to reflect changes I made
- [ ]  I have added new test fixtures as needed to support added tests
- [x]  Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)
- [ ]  I have moved the associated Shortcut story to "Ready for Review"

### Reviewer(s) checklist

- [ ] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.
- [ ] Are there any TODOs in this PR that should be turned into stories?

### CODE REVIEW

**Comment by rebeccabilbro** [comment]
@pdamodaran just gave you access to those repos!

**Comment by pdamodaran** [comment]
I have made the following changes:

- Added client id and client secret to the code
- I created an `img` directory under `static` and moved the images there and verified that the images are rendering now
- I have pushed my code example to `ensign-examples` and `watermill-ensign`.  PRs [here](https://github.com/rotationalio/ensign-examples/pull/1) and [here](https://github.com/rotationalio/watermill-ensign/pull/2).

File: docs/content/examples/data_engineers.md [hunk 1]
```diff
@@ -0,0 +1,609 @@
...
+bookCollapseSection: false
+bookSearchExclude: false
+---
+
+# Ensign for Data Engineers
+
+Hi there! This tutorial is targeted towards Golang data engineers. If you are interested in or currently writing event-driven applications in Go you are in the right place! In this code-driven tutorial we will use the [Watermill-Ensign](https://github.com/rotationalio/watermill-ensign) Golang SDK and [Watermill](https://watermill.io) to call a Weather API and insert the weather information into a PostgreSQL database.  For those who are not familiar with Watermill, refer to my [initial post](https://rotational.io/blog/prototyping-eda-with-watermill/).
+
+TODO: change this link
+If you came here for the code the full example is available [here](https://github.com/rotationalio/ensign-examples/tree/main/go/tweets).
```
Comment (pdamodaran) [inline docs/content/examples/data_engineers.md:16]: we can change this link once the code example is merged into `main`.

**Review by rebeccabilbro — APPROVED** [review]
Great work @pdamodaran ! 
I particularly love the PostgreSQL tie-in &mdash; such a good way to illustrate how Ensign puts data in motion!

**Pull request merged**